src/python/pants/goal/run_tracker.py

```diff
@@ -387,15 +390,51 @@ def store_stats(self):
     if target_data:
       run_information['target_data'] = ast.literal_eval(target_data)
 
+    def get_option(scope_or_scope_with_flag, flag=None):
+      """Looks up a scope in the options parsed by Pants.
+
+      scope_or_scope_with_flag is a string either either of form scope or scope.flag
```

Comment:
I don't love the scope.flag convention. It's confusing, since we're used to reading dotted strings as just scopes, and, hypothetically, it could conflict with some other scope. How about scope^flag or something similar?

Reply:
Yeah, that's actually a lot nicer.